README.md

```diff
@@ -69,6 +69,30 @@ class PaymentsController < ApplicationController
 end
 ```
 
+## You can add more params:
+
+```ruby
+# app/controllers/payments_controller.rb
+class PaymentsController < ApplicationController
+  def create
+    payment = MonopayRuby::Invoices::AdvancedInvoice.new(
+      "https://example.com",                    # your redirect url
+      "https://example.com/payments/webhook"    # your webhook url
```

Comment:
but this code doesn't work, you wrote named arguments in your class, but for this examples provided wrong definition, fix this, please